demos/src/scss/_demos.scss

```diff
@@ -42,19 +45,20 @@ h2 {
 	filter: progid:DXImageTransform.Microsoft.gradient(startColorstr=#{$ie-hex-str},endColorstr=#{$ie-hex-str});
 }
 .o-grid-row {
-	background: #bdcaff;
-	margin-top: 10px;
-	margin-bottom: 10px;
+	background: #f6e9d8;
+	margin-top: 1rem;
+	margin-bottom: 1rem;
 
 	.o-grid-row {
-		background: #dc73ff;
+		background: #cec6b9;
```

Comment:
It doesn't really need to be "exactly on brand", so I wanted to avoid a dependency as much as possible.

Also, we don't usually use devDependencies for building demos, do we?